graveler/committed/apply.go

```diff
@@ -71,6 +71,12 @@ func Apply(ctx context.Context, writer MetaRangeWriter, source Iterator, diffs g
 			haveSource = source.Next()
 		}
 	}
+	if err := source.Err(); err != nil {
+		return err
+	}
+	if err := diffs.Err(); err != nil {
+		return err
+	}
```

Comment:
Consider the shorter
```suggestion
	if source.Err() {
		return source.Err()
	}
	if diffs.Err() {
		return diffs.Err()
	}
```
(and similarly below).

Reply:
python style

Reply:
the first option captures the Err value in case Err can't be inlined or it is more than just a simple return.